[draft] MIP-124 : Validator reward configuration

apenzk opened the pull request:
# Summary

informs how to update the reward rate

[MIP-124](https://github.com/movementlabsxyz/MIP/blob/validator-rewards/MIP/mip-124/README.md)

apenzk committed:
init

apenzk committed:
edit

apenzk committed:
update

apenzk committed:
change dir

apenzk committed:
change headline

apenzk committed:
minor edtis

0xmovses commented on MIP/mip-124/README.md:
```diff
@@ -0,0 +1,237 @@
+
+#### Path 1: Manual Governance Updates
+
+**When Active**: `periodical_reward_rate_decrease_enabled()` is **FALSE** (current system)
```
Note that

Whilst we still have core_resource signer, these proposals will be issued and signed off by the core_resources signer _until_ we have enabled partial OR full governance (partial is recommended initially as it works better for small validator sets). 

We intend to do this _after_ Move2 is out and the Migration is complete on mainnet.

apenzk replied:
added this as a note

0xmovses commented on MIP/mip-124/README.md:
```diff
@@ -0,0 +1,237 @@
+
+#### Path 2: Automatic Rate Decreases
+
+**When Active**: `periodical_reward_rate_decrease_enabled()` is **TRUE** (future system)
```
Would this be preferred? (After Governance is enabled?)

apenzk replied:
either path is ok imo. do you have an intuition which one would be better?

0xmovses reviewed (changes requested):
One ammend, one question. Thank you!

areshand commented on MIP/mip-124/README.md:
```diff
@@ -0,0 +1,237 @@
+
+Update the genesis configuration file to set the target APR to <VALUE>%:
+
+- **File**: `terraform/helm/genesis/values.yaml`
```
We cannot update the genesis configuration.

apenzk replied:
this is just for completeness. So we cover the entire process well

areshand commented:
Since MIP is to get feedbacks on the reward from community, do we also want to talk about what the reward rate is ? or how is it set?

apenzk committed:
updates

apenzk committed:
note on governance

apenzk committed:
bug discovery in path 2

apenzk committed:
change mip number

apenzk committed:
edits to variable names

apenzk committed:
MIP-124: Update reward system docs to target 10% APY (9.53% APR)

- Add APR vs APY section with formulas and continuous compounding limit
- Update all examples to use 9.53% APR for 10% APY target
- Remove duplicate definitions and consolidate configuration sections
- Clarify motivation vs abstract sections

apenzk committed:
Fix math notation for GitHub compatibility

- Replace LaTeX notation with code blocks and Unicode symbols
- GitHub markdown doesn't support \[ \] or \( \) math syntax
